src/scripts/components/animationwidget/gifpanel-grid.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+/*
+ * A table that updates with image
+ * data
+ *
+ * @class ResolutionTable
+ * @extends React.Component
+ */
+export class GifPanelGrid extends React.Component {
+  renderImageSize() {
+    var size = this.props.requestSize;
+    if (!this.props.valid) {
+      return (
+        <div id='gif-size' className='gif-size gif-size-invalid grid-child'>
+          <i className='fa fa-times fa-fw' />
+          <span>{'~' + size + ' MB'}</span>
+        </div>
+      );
+    } else {
+      return (
+        <div id='gif-size' className='gif-size grid-child'>
+          <span>{'~' + size + ' MB'} </span>
+        </div>
+      );
+    }
+  }
+  render() {
+    let imageSize = this.renderImageSize();
+    return (
+      <div className='gif-download-grid'>
+        <div className='grid-child'><span>Start Date: </span></div>
+        <div className='grid-child'><span>{this.props.startDate} </span></div>
+        <div className='grid-child'><span>End Date: </span></div>
+        <div className='grid-child'><span>{this.props.endDate}</span></div>
+        <div className='grid-child'><span>speed</span></div>
```

Comment:
Can you upper case this label and add ":" to each of the labels for consistency

Reply:
When testing I noticed some label inconsistencies. Everything on the left should be upper-case and have a `:`. In the 2nd screenshot, the increment type should probably be upper-cased.

Feels to me like the "Resolution (per pixel)" label and drop down should switch locations for consistency as well.

![screenshot_28](https://user-images.githubusercontent.com/17348884/36861609-54e15da4-1d52-11e8-900b-75ac48e98303.png)
![screenshot_29](https://user-images.githubusercontent.com/17348884/36861610-54f3bbe8-1d52-11e8-8fc1-fb8f07f9ed9d.png)